DECISIONFILE_ENTRY.md

```diff
@@ -0,0 +1,13 @@
+template format for decision files
+
+DATE in YYYY MMM DD
+
+MOTION (mandatory field)
+
+PROPOSER
+
+SUMMARY/DISCUSSION (link to email thread if applicable)
+
+STATUS
+
+DECIDED DATE in YYYY MMM DD
```

Comment:
I'd move this up to just below the date proposed, and for the sake of tidiness rename it DECISION DATE in YYYY MMM DD